duplicate_package_sample.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Specify the source directory and the destination directory
+SOURCE_DIRECTORY="src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0-pre001/Web3.Unity Samples/"
+DESTINATION_DIRECTORY="src/UnityPackages/io.chainsafe.web3-unity/Samples~/Web3.Unity/"
```

Comment:
Is it necessary to have `~` in the Samples folder name?

Reply:
it's not necessary, it's used to exclude samples from the Unity hierarchy, because of the ~ samples can't be found/referenced from project hierarchy and can only be imported.